src/program/librapay.js

```diff
@@ -113,6 +89,105 @@ export async function createGenesis(
   return genesisAccount;
 }
 
+/**
+ * Load a new instance of a Move program on-chain.
+ * Returns the new program account.
+ */
+export async function publishModule(
+  connection: Connection,
+  moduleAccount: Account,
+  path: string,
+): Promise<void> {
+  try {
+    await connection.getAccountInfo(moduleAccount.publicKey);
+  } catch (e) {
+    return await MoveLoader.load(
```

Comment:
If you remove the `await` here, you can avoid having the `await` throw an `UnhandledPromiseException`. This allows the caller of `publishModule` to properly handle an error from `MoveLoader.load(..)`